tests/unit/accounts/test_urls.py

```diff
@@ -5,6 +5,11 @@
 
 from rest_framework.test import APITestCase, APIClient
 
+try:
+    unicode        # Python 2
+except NameError:
+    unicode = str  # Python 3
+
```

Comment:
@cclauss instead of declaring it everywhere, it would make sense to declare it once in `tests/unit/__init__.py`. Does that sound good?

Reply:
That does not work.  😏